appyters/Gene_Age_Trends_Appyter/README.md

```diff
@@ -0,0 +1,7 @@
+Gene-Aging-Trends-Appyter
```

Comment:
The README is in markdown, you should clean it up with that in mind.

You should put a "#" before the start here, check out what it looks like in git rendering:

https://github.com/MaayanLab/appyter-catalog/blob/4630be3777d0eaf78f36f2cb4c5a594fcf4a313c/appyters/Gene_Age_Trends_Appyter/README.md

This is how it will look like on the site. Compared to say: https://github.com/MaayanLab/appyter-catalog/tree/4630be3777d0eaf78f36f2cb4c5a594fcf4a313c/appyters/TCGA_Harmonizome_ETL

Reply:
Got it. Made the change.